Leaf API.postman_collection.json

```diff
@@ -2372,587 +2372,775 @@
+													"disabled": true
+												},
+												{
+													"key": "page",
+													"value": "0",
+													"description": "Pagination parameter for selecting the page of the result (integer)",
+													"disabled": true
+												},
+												{
+													"key": "size",
+													"value": "30",
+													"description": "Pagination parameter that specifies the size of each page (integer)",
+													"disabled": true
+												}
+											]
+										}
 									},
-									{
-										"key": "Authorization",
-										"type": "text",
-										"value": "Bearer {{leaf_token}}"
-									}
-								],
-								"url": {
-									"raw": "https://{{leaf_api_url}}/services/fields/api/growers",
-									"protocol": "https",
-									"host": [
-										"{{leaf_api_url}}"
-									],
-									"path": [
-										"services",
-										"fields",
-										"api",
-										"growers"
-									],
-									"query": [
-										{
-											"key": "leafUserId",
-											"value": "{{leaf_user_id}}",
-											"description": "your Leaf User id",
-											"disabled": true
-										},
-										{
-											"key": "page",
-											"value": "0",
-											"description": "Pagination parameter for selecting the page of the result (integer)",
-											"disabled": true
-										},
-										{
-											"key": "size",
-											"value": "30",
-											"description": "Pagination parameter that specifies the size of each page (integer)",
-											"disabled": true
+									"response": []
+								},
+								{
+									"name": "Get a grower",
+									"request": {
+										"method": "GET",
+										"header": [
+											{
+												"key": "Authorization",
+												"value": "Bearer {{leaf_token}}"
+											}
+										],
+										"url": {
+											"raw": "https://{{leaf_api_url}}/services/fields/api/growers/{{grower_id}}",
+											"protocol": "https",
+											"host": [
+												"{{leaf_api_url}}"
+											],
+											"path": [
+												"services",
+												"fields",
+												"api",
+												"growers",
+												"{{grower_id}}"
+											]
 										}
-									]
+									},
+									"response": []
 								}
-							},
-							"response": []
+							]
 						},
 						{
-							"name": "Get a grower",
-							"request": {
-								"method": "GET",
-								"header": [
-									{
-										"key": "Authorization",
-										"value": "Bearer {{leaf_token}}"
-									}
-								],
-								"url": {
-									"raw": "https://{{leaf_api_url}}/services/fields/api/growers/{{grower_id}}",
-									"protocol": "https",
-									"host": [
-										"{{leaf_api_url}}"
-									],
-									"path": [
-										"services",
-										"fields",
-										"api",
-										"growers",
-										"{{grower_id}}"
-									]
+							"name": "Files",
+							"item": [
```

Comment:
Yeah, I see it can be confusing. It was already here before, we didn't remove it back then. I think we can merge this PR with those endpoints, since it's only adding information of boundaries and they were already there, and we can keep in parallel the discussion of removing/moving it